proto/regen/ecocredit/v1alpha1/types.proto

```diff
@@ -117,4 +117,18 @@ message CreditTypeSeq {
 
   // The sequence number of classes of the credit type
   uint64 seq_number = 2;
-}
\ No newline at end of file
+}
+
+// TagTarget represents the target of a tag.
+message TagTarget {
+  oneof sum {
```

Comment:
would that make sense to also be able to target credit types?

Reply:
Credit types should be so few that I'm not sure there's a value... @clevinson ?